Assets/MixedRealityToolkit/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -93,22 +91,19 @@ public float WheelSpeed
         /// </summary>
         public MouseController Controller { get; private set; }
 
-        private void ReadProfile()
```

Comment:
Why did you remove this? Will Cursor and wheel speed still be initialized?

Reply:
Not deleted, just moved to end of file. See full file change